app/assets/javascripts/validation.coffee

```diff
@@ -891,6 +892,23 @@ $(document).ready ->
 
           errors.push(error)
 
+  validatePotentialActionUrlTemplate = (json, errors) ->
+    # Source of following regular expression is https://regex101.com/r/DstcXC/1/
+    # which is pointed from https://stackoverflow.com/questions/29494608/regex-for-uri-templates-rfc-6570-wanted
+    # This method should be removed after the regex is added to the umm-t schema.
+    # Ticket for adding regex to umm-t schema is https://bugs.earthdata.nasa.gov/browse/ECSE-1117
```

Comment:
Not a huge deal, but might be helpful to have the MMT ticket here instead, as that is where the actual change should be made.